internal/login/login.go

```diff
@@ -126,6 +127,20 @@ func UserCredentialsLogin(p LoginParameters) {
 	return
 }
 
+
+func CredentialsLogout(p LoginParameters) {
+	twitchClientCredentialsURL := fmt.Sprintf(`https://id.twitch.tv/oauth2/revoke?client_id=%s&token=%s`, p.ClientID, p.Token)
+
+	_, err := loginRequest(http.MethodPost, twitchClientCredentialsURL, nil)
+	if err != nil {
+		log.Fatal(err.Error())
+		return
+	}
+
+	println("Done.")
```

Comment:
It might be nice here to sate that the token was revoked specifically vs. "Done".

Reply:
Might also be adding an additional check for status code "200" of the response, so when stating "Successfully revoked" the code can actually be sure it DID get on OK from the API.